analyzer-models/src/main/java/ffb/analyzer/models/espn/Player.java

```diff
@@ -0,0 +1,177 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import ffb.analyzer.models.espn.deserializers.PlayerRankingsDeserializer;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Entity representing a player.
+ */
+public class Player extends EspnEntity<Player> {
+
+    /**
+     * Enum for a player's position.
+     */
+    public enum Position {
+        QB(1),
+        RB(2),
+        WR(3),
+        TE(4),
+        K(5),
+        DEFENSE(16);
+
+        private static final Map<Integer, Position> POSITION_BY_ID;
```

Comment:
Don't think we need this -- We didn't in `LeagueType` to get the enum from an `int`.

Reply:
We need this because the values aren't sequential. So in order to correctly grab defense, we need to have that map.